Integrations/integration-Cisco-umbrella.yml

```diff
@@ -0,0 +1,885 @@
+    }
+
+    // "one_two" to "One Two"
+    var keyToTableKey = function(s) {
+        return s && s.replace(/_/g, ' ')
+                .replace(/\w+/g, function(w){return w[0].toUpperCase() + w.slice(1).toLowerCase();});
+    }
+
+    // "one_two" to "OneTwo"
+    var keyToContextKey = function(s) {
+        return s && keyToTableKey(s).replace(/\s+/g, '');
+    }
+
+    var createTableEntry = function (name, contents, context, headers) {
+        return {
+            // type
+            Type: entryTypes.note,
+             // contents
+            ContentsFormat: formats.json, Contents: contents,
+            // human-readable
+            ReadableContentsFormat: formats.markdown, HumanReadable: tableToMarkdown(name, contents, headers),
+            // context
+            EntryContext: context
+        };
+    }
+
+    var startArg = (args.start) ? args.start : '';
+
+    var sendRequest = function(requestUrl) {
+        var res = http(
+            requestUrl,
+            {
+                Method: 'GET', // Can be POST, PUT, DELETE, HEAD, OPTIONS or CONNECT
+                Headers: {
+                    Authorization: ['Bearer ' + apitoken],
+                    Accept: ['application/json']
+                }
+            },
+            insecure,
+            proxy
+        );
+
+
+        if (!res || res.StatusCode < 200 || res.StatusCode >= 300) {
+            var errorString = '';
+            var body = JSON.parse(res.Body);
+
+            if (body && body.error == 'unauthorized') {
+                errorString = '\n\nError: Unauthorized check your API token\n';
+            }
+            errorString += '\nRequest Failed'
+                + '\nRequestUrl: ' + requestUrl
+                + '\nStatus code: ' + res.StatusCode
+                + '\nResponse: ' + JSON.stringify(res);
+            throw errorString;
+        }
+
+        var resBody = JSON.parse(res.Body);
+        if (resBody === undefined) {
+           throw 'Request Failed, returned response with no body';
+        }
+        return resBody;
+    };
+
+    var getDomainCategorization = function(domain) {
+        domain = domainWithoutProtocol(domain);
+        var url = baseURL + '/domains/categorization/' + encodeURIComponent(domain) + '?showLabels';
+
+        var res = sendRequest(url);
+        var categorization = res && res[domain];
+
+        // build context & table
+        var domainContext = {
+            Name: domain
+        };
+        var table = {};
+        if (categorization) {
+            if (categorization.status !== undefined) {
+                table['Status'] = categorization.status
+            }
+            if (categorization.content_categories && categorization.content_categories.length > 0) {
+                var contentCategories = categorization.content_categories.join(',');
+                table['Content Catecories'] = contentCategories;
+                domainContext.ContentCategory = contentCategories;
+            } else {
```

Comment:
don't use if else, just do
table['Content Catecories'] = 'No Content Categories Were Found';
followed by the if that will override it